app/src/main/java/com/isc/hermes/controller/LocationPermissionsController.java

```diff
@@ -0,0 +1,89 @@
+package com.isc.hermes.controller;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+
```

Comment:
It also removes unused spaces.

Reply:
@VillalobosGarciaGaby  This recommendation is already satisfied.